lib/tuya.js

```diff
@@ -419,6 +419,11 @@ const dataPoints = {
     childLock: 7,
     coverChange: 7,
     runningState: 14,
+    /**
+     * Data Transfer Type: Send and Report
+     * Data Type: Bool
+     */
+    muffling: 16,
```

Comment:
Why was this added? (I do not see it being used)

Reply:
I was in contact with the vendor to check what DPs are supported this device - and they claimed that it is also possible to mute the device via DP16 - but I could not verify this function with my three devices. It seems not to be working. Anyhow - as this is a common data point I kept it in the code for future useage to make it easier to identify the dunction of data points. But we can also remove this if you think we should not keep it in (as it is currently not used).

Reply:
Please remove it and add it to https://github.com/Koenkk/zigbee-herdsman-converters/pull/5613/files#diff-7ae23d3c13c2d7f7e37106590684c791aba8093f8be813b7c74a4f1484017c3bR2730 as a comment.